packages/composer/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/composer/composer
+TERMUX_PKG_DESCRIPTION="Composer is a tool for dependency management in PHP."
+TERMUX_PKG_LICENSE=MIT
+TERMUX_PKG_VERSION=2
```

Comment:
That version number is fake.

Reply:
All of them are resolved.

Reply:
Home Page url is changed to 
[https://github.com/archlinux/svntogit-packages/blob/packages/composer/trunk/PKGBUILD](https://github.com/archlinux/svntogit-packages/blob/packages/composer/trunk/PKGBUILD)

Reply:
If you think my code formation is bad.
Please format,I code from my phone.
I use tab for your comments.
But Idk it work or not in your IDE.